### PR TITLE
logging: hide entries when debuglog is set to false and add missing worker names (PROJQUAY-6562)

### PR DESCRIPTION
**Pull request by sayalibhavsar**
fixing log discrepancies as per reviewer's suggestion from https://github.com/quay/quay/pull/3160

### CODE REVIEW

File: util/migrate/allocator.py [hunk 1]
```diff
@@ -10,11 +11,18 @@
...
-else:
-    logging.basicConfig(level=logging.INFO)
+
+# Define the logging format to include worker_name and stream_type
+log_format = "%(asctime)s [%(process)d] [%(levelname)s] [%(name)s] %(message)s"
+
+# Set the logging level and format based on DEBUGLOG
+
+# Set up logging
+logging.basicConfig(
```
Comment (Marcusk19) [inline util/migrate/allocator.py:21]: Should we be using [basicConfig](https://docs.python.org/3/library/logging.html#logging.basicConfig) here? It seems like this configures the root logger which might not be what we want

Comment (sayalibhavsar) [reply util/migrate/allocator.py:21]: You are right. logging.basicConfig affects all logging calls and we want granular control for module util.migrate.allocator. Doing it with [setLevel](https://docs.python.org/3/library/logging.html#logging.Logger.setLevel) with handlers and formatters

**Review by Marcusk19 — APPROVED** [review]
lgtm

**Pull request merged**